src/main/resources/languages/zh-CN/messages.yml

```diff
@@ -124,6 +124,8 @@ guide:
       wiki: '&3Wiki 编辑者'
       resourcepack: '&c材质制作者'
       translator: '&9翻译者'
+actionbar:
+  radiation: '&6辐射暴露等级：&c%level%&7/&100'
```

Comment:
`&100` will turn into color code &1 and 00

Reply:
not sure who can change this - @ybw0014 can you (or know who?)

Reply:
Im not language moderator.

Reply:
Re-approved correct one.